src/commands/troubleshoot.ts

```diff
@@ -0,0 +1,54 @@
+import { Args, Command, Config, Flags } from "@oclif/core";
+import Troubleshooter from "../run/troubleshooter.js";
+import load from "../helpers/config-loader.js";
+
+export default class Troubleshoot extends Command {
```

Comment:
Split into two commands: `Troubleshoot` and `Report` - Troubleshoot identifies the errors in hope user completes these errors, report produces an artifact detailing the environment to help provide support